#include <iostream>
using namespace std;

#include "Pilas.h"

int main(){

	try{
		Pilas pil;
		pil.sacarexit
		();
		
		
	}
	catch (Pilas::errorPilas ex1){
		cout<<"ERROR: En el metodo "<<ex1.observa<<endl;
	}
	return 0;
}	
	